x-pack/plugins/apm/typings/apm_rum_react.d.ts

```diff
@@ -3,5 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-declare module '@elastic/apm-rum-react';
+declare module '@elastic/apm-rum-react' {
+  export function indexManagementStore(): any;
```

Comment:
this variable is not part of `@elastic/apm-rum-react. 

https://github.com/elastic/apm-agent-rum-js/blob/3e3c73e2836d89663657e5e63305d518838d46d6/packages/rum-react/src/index.js#L33

Reply:
Thanks, I'll make sure to remove it in other PR. The goal of this PR is to make sure `master` matches the `7.x` branch for this file in order to minimize backport conflicts.

Reply:
actually i think i can do it in this PR as long as i backport it. I was avoiding a backport, but in this case i think it's fine.